ios/history.md

```diff
@@ -1,5 +1,8 @@
 # Keyman for iPhone and iPad Version History
 
+## 2019 01-04 11.0.302 beta
+* Improves loading efficiency of system keyboard, helping prevent related crashes (#1169, #1170, #1172)
```

Comment:
The history entry should reference the PR rather than the issue.

Reply:
I thought we'd transitioned to referencing the issues?  Or is that only for certain platforms?  I may have to correct some other history entries in the near future, then...

Reply:
No, we reference PRs, have done for about half a year at least (although I think you may have been away at the time, sorry)